cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -375,4 +378,44 @@ private static List<String> validateComputeResources(ResourceRequirements resour
         }
         return errors;
     }
-}
+
+    /**
+     * Append list configuration values or create a new list configuration if missing.
+     * A list configuration can contain a comma separated list of values.
+     * Duplicated values are removed.
+     *
+     * @param kafkaConfig Kafka configuration.
+     * @param key List configuration key.
+     * @param values List configuration values.
+     */
+    public static void createOrAddListConfig(AbstractConfiguration kafkaConfig, String key, String values) {
```

Comment:
These are not really model utils from my point of view and should not be here. (Plus as I said, you should not need these things in the first place.)

Reply:
Can you suggest a better place please? Thanks

Reply:
Possibly in the configuration classes themself? But as I said in the other place, I do not think you should be adding to it in the first place?

Reply:
Ok, I will use AbstractConfiguration to host that method. Thanks for the suggestion.

Reply:
I have moved `createOrAddListConfig` whilst awaiting the outcome of the conversation here https://github.com/strimzi/strimzi-kafka-operator/pull/11570#discussion_r2164751526